[2025-07-18T04:20:34.964+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: batch_etl_pyspark.run_pyspark_etl scheduled__2025-07-17T00:00:00+00:00 [queued]>
[2025-07-18T04:20:35.016+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: batch_etl_pyspark.run_pyspark_etl scheduled__2025-07-17T00:00:00+00:00 [queued]>
[2025-07-18T04:20:35.018+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-07-18T04:20:35.019+0000] {taskinstance.py:1280} INFO - Starting attempt 2 of 2
[2025-07-18T04:20:35.021+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-07-18T04:20:35.125+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): run_pyspark_etl> on 2025-07-17 00:00:00+00:00
[2025-07-18T04:20:35.151+0000] {standard_task_runner.py:55} INFO - Started process 336 to run task
[2025-07-18T04:20:35.170+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'batch_etl_pyspark', 'run_pyspark_etl', 'scheduled__2025-07-17T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/batch_etl_dag.py', '--cfg-path', '/tmp/tmpxdcbsiqz']
[2025-07-18T04:20:35.181+0000] {standard_task_runner.py:83} INFO - Job 7: Subtask run_pyspark_etl
[2025-07-18T04:20:35.923+0000] {task_command.py:388} INFO - Running <TaskInstance: batch_etl_pyspark.run_pyspark_etl scheduled__2025-07-17T00:00:00+00:00 [running]> on host f5e43c0cbf33
[2025-07-18T04:20:36.544+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=batch_etl_pyspark
AIRFLOW_CTX_TASK_ID=run_pyspark_etl
AIRFLOW_CTX_EXECUTION_DATE=2025-07-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-07-17T00:00:00+00:00
[2025-07-18T04:20:36.557+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2025-07-18T04:20:36.565+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'docker exec spark spark-submit /opt/bitnami/spark/scripts/batch_etl_day.py']
[2025-07-18T04:20:36.612+0000] {subprocess.py:86} INFO - Output:
[2025-07-18T04:20:36.750+0000] {subprocess.py:93} INFO - Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
[2025-07-18T04:20:36.776+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-07-18T04:20:36.842+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-07-18T04:20:36.860+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=batch_etl_pyspark, task_id=run_pyspark_etl, execution_date=20250717T000000, start_date=20250718T042034, end_date=20250718T042036
[2025-07-18T04:20:36.999+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 7 for task run_pyspark_etl (Bash command failed. The command returned a non-zero exit code 1.; 336)
[2025-07-18T04:20:37.057+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2025-07-18T04:20:37.165+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
